src/libraries/System.Private.CoreLib/src/System/Globalization/CultureData.Browser.cs

```diff
@@ -10,13 +10,63 @@ namespace System.Globalization
     internal sealed partial class CultureData
     {
         private const int CULTURE_INFO_BUFFER_LEN = 50;
+        private const int LOCALE_INFO_BUFFER_LEN = 80;
+
+        private static CultureData JSInitLocaleInfo(string? localeName, CultureData culture)
```

Comment:
IIUC the method header can be simplified 
- the `localeName` is `culture._sName`
- no need to return since the method overrides `culture` param

```suggestion
        private static void JSInitLocaleInfo(CultureData culture)
```